### ISSUE TITLE
Can't compile any subdirectory with Maven

### ISSUE THREAD

**Issue by thomaskrause** [hexatomic/hexatomic#75]
**Describe the bug**
It is no longer possible to execute any Maven command in a non-root folder of the project. The error 
```
Could not resolve target platform specification artifact org.corpus_tools:org.corpus_tools.hexatomic.target:target:0.2.0-SNAPSHOT
``` 
is thrown

**To reproduce the bug**
1. Go to hexatomic/bundles directory
2. Execute `mvn install`

**Expected behavior**
Should compile the sub-projects (and nothing more)

**Desktop (please complete the following information):**
 - Operating System: Ubuntu 19.10
 - Hexatomic Version: 0.2.0-SNAPSHOT
 - Java Version: OpenJDK 8

**Additional context**

Bisecting tells me the bug was introduced in commit b01598da222ea8aedda7470c1cbd037829d8cb44

**Comment by thomaskrause** [comment]
@sdruskat Can you have a look at it. Maybe a project version was not updated?

**Comment by thomaskrause** [comment]
Problem is that we need to call 
```
mvn install
```
in the root folder once and not only `mvn verify` like in most parts of the documentation to allow building sub-modules separately.

**Comment by thomaskrause** [comment]
Closed with Hotfix/0.1.3
